repo2docker/__main__.py

```diff
@@ -434,12 +435,13 @@ def make_r2d(argv=None):
         sys.exit(1)
 
     if args.build_memory_limit:
-        # if the string only contains numerals we assume it should be an int
-        # and specifies a size in bytes
-        if args.build_memory_limit.isnumeric():
-            r2d.build_memory_limit = int(args.build_memory_limit)
-        else:
-            r2d.build_memory_limit = args.build_memory_limit
+        # We no longer support build_memory_limit, it must be set in the builder instance
+        print("--build-memory-limit is no longer supported", file=sys.stderr)
+        print(
+            "Use `docker buildx create` to create a custom builder with appropriate memory limits instead",
```

Comment:
```suggestion
            Check your build engine documentation to set memory limits. Use `docker buildx create` if using the default builder to create a custom builder with appropriate memory limits,
```
I didn't notice this message was duplicated- otherwise this looks fine to merge!